app/workers/set_tenant_id_worker.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class SetTenantIdWorker < ApplicationJob
+
+  def perform(provider_batch)
+    provider_batch.each do |provider|
+      # Set the master field to false
+      provider.master = false
+      provider.save!
```

Comment:
Why not `provider.update(master: false)`?

Reply:
will this raise on error?

Reply:
According to docs, it will not raise on error.

Reply:
I would use 

```ruby
provider.update_column(:master, false)
```